src/it/test-skip-property/verify.groovy

```diff
@@ -0,0 +1,9 @@
+import org.codehaus.plexus.util.FileUtils;
+
+String log = FileUtils.fileRead(new File(basedir, "build.log"));
+
+if (!log.contains("snyk test skipped")) {
+    throw new Exception("skip message not found");
```

Comment:
Should also add a failure scenario (like a vulnerability) to ensure CLI doesn't execute.